### ISSUE TITLE
Bazel: allow cc deps in go rules [phab:D337]

### ISSUE THREAD

**Issue by kythebot** [google/kythe#559]
_Created by **schroederc** at 2015-06-12 16:55:12_:

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-06-12 17:02:34_:

Remove libleveldb from deps

**Comment by kythe** [comment]
_**zarko** wrote at 2015-06-12 17:03:41_:

On both OS X and Linux, I see:

```
$ bazel build //...
ERROR: /Users/zarko/kythe/third_party/go/BUILD:45:15: in cc_deps attribute of go_library rule //third_party/go:levigo: '//third_party/leveldb:leveldb' does not have mandatory provider 'cc'.
ERROR: Analysis of target '//kythe/go/serving/search:search_test' failed; build aborted.
```

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-06-12 17:04:48_:

>>! In (earlier comment), zarko wrote:
> On both OS X and Linux, I see:
> 
> ```
> $ bazel build //...
> ERROR: /Users/zarko/kythe/third_party/go/BUILD:45:15: in cc_deps attribute of go_library rule //third_party/go:levigo: '//third_party/leveldb:leveldb' does not have mandatory provider 'cc'.
> ERROR: Analysis of target '//kythe/go/serving/search:search_test' failed; build aborted.
> ```

You likely need to update Bazel.

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-06-15 17:25:04_:

Use absolute paths in include flags

**Comment by kythe** [comment]
_**zarko** wrote at 2015-06-15 17:32:50_:

Checked this on OS X too.

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2015-06-15 17:32:50_

**Comment by kythe** [comment]
_**schroederc** closed via commit 8be260e05f98710559d400a9bbdc07fde2d9cd61 at 2015-06-15 17:37:07_
